src/test/java/teammates/test/driver/GaeSimulation.java

```diff
@@ -206,9 +207,11 @@ public void loginAsStudentInstructor(String userId) {
      * @param method The request method
      * @param body The request body
      * @param parts The request parts
+     * @param cookieValue The request's cookie value
      * @param params Parameters that appear in a HttpServletRequest received by the app
      */
-    public Action getActionObject(String uri, String method, String body, Map<String, Part> parts, String... params) {
+    public Action getActionObject(String uri, String method, String body, Map<String, Part> parts,
+                                  String cookieValue, String... params) {
```

Comment:
`String cookieValues` should be `List<Cookie> cookies`?

The method is supposed to be generic. There are many cookies besides CSRF.